llvm/test/CodeGen/X86/insert.ll

```diff
@@ -0,0 +1,34 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 4
+;RUN: llc < %s -mtriple=x86_64-unknown-unknown | FileCheck %s --check-prefixes=X64
```

Comment:
space between ; and RUN.

Reply:
Could we use the default check prefix? `X64` is kind of misleading.

Reply:
Done.

x64 is used elsewhere too. Anyway change it to x86_64 and I386 for clarity.